client/app/queue/cavcDashboard/CavcDashboardIssuesSection.jsx

```diff
@@ -33,8 +34,15 @@ const olStyling = css({
 });
 
 const CavcDashboardIssue = (props) => {
-  const [disposition, setDisposition] = useState('Select');
-  const { issue, index, dispositions } = props;
+  const { dispositions, issue, index } = props;
+  const issueDisposition = dispositions?.filter((dis) => {
+    return dis.request_issue_id === issue.id;
+  });
```

Comment:
```suggestion
  const issueDisposition = dispositions?.filter((dis) => dis.request_issue_id === issue.id);
```
What happens if more than one dispositions meets this requirement?

Should this happen in CavcDashboardIssuesSection instead so this component is just focused on a single issue and not concerned with other issues?